dart/lib/src/transport/http_transport.dart

```diff
@@ -39,20 +44,24 @@ class HttpTransport implements Transport {
   }
 
   @override
-  Future<SentryId> send(SentryEvent event) async {
-    final data = event.toJson();
+  Future<SentryId> sendSentryEvent(SentryEvent event) async {
```

Comment:
Duplicated comment but: I believe this method shouldn't exist.

Reply:
Mny unit tests rely on this, as we are injecting a mock and checking the event. Since we don't have the ability to deserialize an Event from a Envelope and didn't want to re-implement a bunch of events, we choose to do is this way for now.